wesnoth-thot.po

```diff
@@ -730,9 +723,8 @@ msgid ""
 "Now that is a sight still strange to my eyes — an orc fighting for the "
 "Northern Alliance."
 msgstr ""
-"Jak zvláštní pohled se mi naskýtá — skřet bojuje na straně Severní aliance."
+"Jak zvláštní pohled se mi naskýtá - skřet bojuje na straně Severní aliance."
```

Comment:
správně opravujete dlouhou pomlčku na nesprávný spojovník; v češtině se používá krátká pomlčka - vizte znovu [mou stránku](https://emer.cz/michal/wesnoth.html)
```suggestion
"Jak zvláštní pohled se mi naskýtá – skřet bojuje na straně Severní aliance."
```

Reply:
Asi bych potřeboval klávesovou zkratku. Respektive možná by to obecně v návodu chtělo specifikovat ty české znaky (uvozovky, pomlčky apod.) i pomocí příslušných klávesových zkratek.

Reply:
Přidal jsem na [svou stránku](https://emer.cz/michal/wesnoth.html) [tabulku znaků](https://emer.cz/michal/wesnoth.html#tabulka%20znak%C5%AF) s odkazy.